raw_result_diff.py

```diff
@@ -0,0 +1,198 @@
+# Copyright (c) 2021, Matthew Olsson <mattco@serenityos.org>
+#
+# SPDX-License-Identifier: MIT
+
+import json
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Tuple
+
+from main import TestResult
+from main import EMOJIS
+
+
+def read_result_files(old_path: Path, new_path: Path) -> Tuple[float, dict, dict]:
+    with old_path.open() as f:
+        old_results = json.loads(f.read())
+    with new_path.open() as f:
+        new_results = json.loads(f.read())
```

Comment:
We already get `Path`s so this can be:

```suggestion
    old_results = json.loads(old_path.read_text())
    new_results = json.loads(new_path.read_text())
```